contrib/executor/jmeterd/README.md

```diff
@@ -4,19 +4,21 @@
 [![Go Reference](https://pkg.go.dev/badge/github.com/kubeshop/testkube-executor-jmeter.svg)](https://pkg.go.dev/github.com/kubeshop/testkube-executor-jmeter)
 [![License](https://img.shields.io/github/license/kubeshop/testkube-executor-jmeter)]()
 
-# JMeter Executor
+# JMeterd Executor
+An Extension of Jmeter Executor which can run the Jmeter Tests in distributor environment.
 
 ## What is an Executor?
 
 Executor is nothing more than a program wrapped into Docker container which gets JSON (testube.Execution) OpenAPI based document as an input and returns a stream of JSON output lines (testkube.ExecutorOutput),
 where each output line is simply wrapped in this JSON, similar to the structured logging idea.
 
+
 ## Intro
 
-It's basic JMeter executor able to run simple JMeter scenarios writer in JMX format.
+It's basic JMeterd executor able to run simple JMeter scenarios in distributed environments.
 Please define your JMeter file as file (string, or git file). 
 
-Project directory is not implemented yet.
+**Before Getting into Jmeterd executor we need to learn about the Jmeter Executor**
```

Comment:
```suggestion
**Basic understanding of Jmeter executor is recommended before running this distributed executor**
```

Also, I'm thinking if this is necessary at all

Reply:
Yeah it's not necessary since we have not removed anything which was present in JMeter README file. So Removed.